config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -1611,13 +1587,35 @@ def generate_presubmits_e2e():
             }
         ),
         presubmit_test(
-            name="pull-kops-e2e-aws-upgrade-k126-ko1270b2-to-k127-kolatest-karpenter",
+            name="pull-kops-e2e-aws-upgrade-k126-ko126-to-k127-kolatest-karpenter",
+            optional=True,
+            distro='u2204arm64',
+            networking='cilium',
+            k8s_version='stable',
+            kops_channel='alpha',
+            test_timeout_minutes=120,
+            scenario='upgrade-ab',
+            extra_flags=[
+                "--instance-manager=karpenter",
+                "--master-size=c6g.xlarge",
+            ],
+            env={
+                'KOPS_FEATURE_FLAGS': "Karpenter",
+                'KOPS_VERSION_A': "1.26",
+                'K8S_VERSION_A': "v1.26.0",
+                'KOPS_VERSION_B': "latest",
+                'K8S_VERSION_B': "v1.27.0",
+                'KOPS_SKIP_E2E': '1',
+                'KOPS_CONTROL_PLANE_SIZE': '3',
+            }
+        ),
+        presubmit_test(
+            name="pull-kops-e2e-aws-upgrade-k126-ko127-to-k127-kolatest-karpenter",
```

Comment:
```suggestion
            name="pull-kops-e2e-aws-upgrade-k127-ko127-to-k127-kolatest-karpenter",
```
?

Reply:
This is intentional, want to see some nodes rolling also.